simd/loongson/jsimd_mmi.h

```diff
@@ -33,6 +33,34 @@
 
 /* Common code */
 
+#if defined(_ABI64) && _MIPS_SIM == _ABI64
+#define mips_reg int64_t
+# define PTR_ADDU       "daddu "
+# define PTR_ADDIU      "daddiu "
+# define PTR_ADDI       "daddi "
+# define PTR_SUBU       "dsubu "
+# define PTR_L          "ld "
+# define PTR_LI         "dli "
+# define PTR_S          "sd "
+# define PTR_SRA        "dsra "
+# define PTR_SRL        "dsrl "
+# define PTR_SLL        "dsll "
+# define MTC1           "dmtc1 "
+#else
+#define mips_reg int32_t
+# define PTR_ADDU       "addu "
+# define PTR_ADDIU      "addiu "
+# define PTR_ADDI       "addi "
+# define PTR_SUBU       "subu "
+# define PTR_L          "lw "
+# define PTR_LI         "li "
+# define PTR_S          "sw "
+# define PTR_SRA        "sra "
+# define PTR_SRL        "srl "
+# define PTR_SLL        "sll "
+# define MTC1           "mtc1 "
+#endif
+
```

Comment:
@zhanglixia-hf Can you explain the purpose of this code?

Reply:
How does one test 32-bit code on Loongson?  The test system appears to be 64-bit-only and does not support the `-m32` argument to GCC.

Reply:
Sorry，we mainly maintain 64-bit systems. At present, I don't have a suitable 32-bit system. The 32-bit compiler option is -mabi=32, but my current system cannot compiles some 32 packages .  So I may be give you after the Spring Festival.